lib/messages.ts

```diff
@@ -1,22 +1,32 @@
-import { Config, RuleBundle } from "./types";
+import { Config, ConsentState, RuleBundle } from "./types";
 
 export type BackgroundMessage =
-  InitResponseMessage
+  | InitResponseMessage
   | EvalResponseMessage
   | OptOutMessage
   | OptInMessage
-  | SelfTestMessage;
+  | SelfTestMessage
+  | ReportMessage;
 
 export type ContentScriptMessage =
-  InitMessage
+  | InitMessage
   | EvalMessage
   | DetectedMessage
   | FoundMessage
   | OptOutResultMessage
   | OptInResultMessage
   | SelfTestResultMessage
   | DoneMessage
-  | ErrorMessage;
+  | ErrorMessage
+  | ReportResponseMessage;
```

Comment:
What do you envision being added there? These messages are out-of-band of the flow described in that document, and unidirectional, so the benefit of showing it on a flow diagram is limited.

Reply:
I just thought it'd be useful to mention the `ReportMessage` there because it is sent from the CS to the BS. It's not essential to the functionality, but probably useful if you're implementing a background script.